### DISCUSSION TITLE
Let the owner add as many maps as they want in their own multi room

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23687] [Ideas] [answered]
I think only the owner of a room should be allowed to add as many beatmaps as they wish because it's their room, regardless of whether the Queue Mode is "Host only", "All players", or "All players (round robin)".
Usually what you have to do is change the Queue Mode to "Host only" and then add your maps and then change it back to "All players". This is inconvenient for the other room guests and you as the owner don't actually want to disallow them from adding maps.

**Comment by peppy** [comment]
That's arguable. Right now, the host gets equal privileges to other users in the latter two modes, which kinda makes sense if you don't want the host hogging the playlist.

I probably agree though.

**Reply by ghost** [reply]
Maybe add an option that the owner can enable to remove the limit of beatmaps he can add while in a mode other than "Host only"?

**Reply by ghost** [reply]
If the owner wants to add more than 3 beatmaps then they should just be allowed to do that. Having it behind an option would only complicate things for no reason IMO.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu-server-spectator/pull/173
